mobius/src/geometry/circular_arc.rs

```diff
@@ -4,12 +4,12 @@ use thiserror::Error;
 
 use crate::Complex;
 
-use super::{circle::Circle, ArcAngles, ArcAnglesParseError, ArcDirection, DirectedEdge, Geometry};
+use super::{circle::Circle, ArcAngles, ArcAnglesError, ArcDirection, DirectedEdge, Geometry};
 
 #[derive(Debug, Error)]
 pub enum CircularArcError {
```

Comment:
Yeah... the more I think about this, the more I'd like to see just a single `GeometryError`